.github/workflows/run-checks.yml

```diff
@@ -0,0 +1,32 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-nodejs
+
```

Comment:
Chore: comment is no longer accurate (particularly about different versions of node)